ios/browser/api/ai_chat/ai_chat.mm

```diff
@@ -72,82 +69,60 @@ - (void)dealloc {
   web::GetUIThreadTaskRunner({})->PostTask(
       FROM_HERE,
       base::BindOnce(
-          ^(decltype(driver_) driver, decltype(ai_chat_metrics_) metrics) {
-            driver.reset();
-            metrics.reset();
+          ^(decltype(current_content_) current_content) {
+            current_content.reset();
           },
-          std::move(driver_), std::move(ai_chat_metrics_)));
+          std::move(current_content_)));
+}
+
+- (void)createNewConversation {
```

Comment:
If this function is never called in Swift, so we can remove it from the header (it doesn't need to be exported).

Reply:
It is called from swift: https://github.com/brave/brave-core/blob/003ef36fa17d7693d9e2dcd07bdfccfb1266be95/ios/brave-ios/Sources/AIChat/ModelView/AIChatViewModel.swift#L176